From: Christopher Huhn <C.Huhn@gsi.de>
Date: Mon, 16 May 2022 16:12:55 +0200
Subject: Disable skb_frag_off

skb_frag_off has been added to 4.19.201, must not be defined here.
Cf. https://sourceforge.net/p/e1000/bugs/692/
---
 src/kcompat.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/kcompat.h b/src/kcompat.h
index be40036..80e5146 100644
--- a/src/kcompat.h
+++ b/src/kcompat.h
@@ -7055,7 +7055,7 @@ int _kc_flow_block_cb_setup_simple(struct flow_block_offload *f,
 #endif /* 5.3.0 */
 
 /*****************************************************************************/
-#if (LINUX_VERSION_CODE < KERNEL_VERSION(5,4,0))
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(4,19,201))
 #if (!(RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(8,2)) && \
      !(SLE_VERSION_CODE >= SLE_VERSION(15,2,0)))
 static inline unsigned int skb_frag_off(const skb_frag_t * frag)
@@ -7074,9 +7074,9 @@ static inline void skb_frag_off_add(skb_frag_t * frag, int delta)
 #if (SLE_VERSION_CODE >= SLE_VERSION(15,2,0))
 #define HAVE_NDO_XSK_WAKEUP
 #endif /* SLES15sp2 */
-#else /* >= 5.4.0 */
+#else /* >= 4.19.201 */
 #define HAVE_NDO_XSK_WAKEUP
-#endif /* 5.4.0 */
+#endif /* 4.19.201 */
 
 /*****************************************************************************/
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(5,6,0))
